back/admin.http

```diff
@@ -0,0 +1,23 @@
+GET http://localhost:8000/api/camping
+
+###
```

Comment:
this line deserves a true comment (not just "###") to precise what is done by this request :)